### PR TITLE
feat: provide links to original docs

### PR DESCRIPTION
**Pull request by RafalWilinski**
Closes https://github.com/Stedi/mappings/issues/241
Screenshot 2022-04-05 at 15 37 06

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Welcome to Registry contributing guide
+
+Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on [stedi.com/registry/json-schemas](https://www.stedi.com/registry/json-schemas) :sparkles:. 
+
+In this guide you will get an overview of the contribution workflow from creating a provider, creating a PR, reviewing, and merging the PR.
+
+### Adding a provider
+
+To add a new provider, please create a new file in the `src/providers` repository. The file name should be the provider name + `.ts` suffic, and the file should contain the logic to fetch the provider data.
```
Comment (mthenw) [inline CONTRIBUTING.md:9]:
```suggestion
To add a new provider, please create a new file in the `src/providers` repository. The file name should be the provider name + `.ts` suffix, and the file should contain the logic to fetch the provider data.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+If your provider exposing their API using OpenAPI or Swagger, you can use following template to automate this process:
```
Comment (mthenw) [inline CONTRIBUTING.md:11]:
```suggestion
If your provider exposes their API using OpenAPI or Swagger, you can use the following template to automate this process:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      description:
+        "My Company is a company...",
+      versions: ["v1"],
+      baseUrl: "https://docs.my-company.com/meta/v1/openapi.json",
+    });
+  }
+}
+```
+
+Please also add your provider to the `src/providers/index.ts` file.
```
Comment (mthenw) [inline CONTRIBUTING.md:30]: What about providers.json?

Comment (RafalWilinski) [reply CONTRIBUTING.md:30]: It's autogenerated, logic in the previous PR

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,14 +2,15 @@
...
 
-This guide will give you an overview of the contribution workflow from creating a provider, creating a PR, reviewing, and merging the PR.
+In this guide you will get an overview of the contribution workflow from creating a provider, creating a PR, reviewing, and merging the PR.
 
 ### Adding a provider
 
-Please create a new file in the `src/providers` repository to add a new provider. The file name should be the provider name + `.ts` suffix, and the file should contain the logic to fetch the provider data.
+To add a new provider, please create a new file in the `src/providers` repository. The file name should be the provider name + `.ts` suffix, and the file should contain the logic to fetch the provider data.
 
 If your provider is exposing their API using OpenAPI or Swagger, you can use the following template to automate this process:
```
Comment (WojciechMatuszewskiStedi) [inline CONTRIBUTING.md:11]: Minor nit: what if my provider is not exposing either OpenAPI or Swagger?

**Review by WojciechMatuszewskiStedi — APPROVED** [review]

**Pull request merged**